server/homebrew.model.js

```diff
@@ -16,6 +16,7 @@ const HomebrewSchema = mongoose.Schema({
 	renderer    : { type: String, default: '' },
 	authors     : [String],
 	published   : { type: Boolean, default: false },
+	language   	: { type: String, default: 'en' },
```

Comment:
Make similar changes for the Google Drive process. Look in `googleActions.js` at the `updateGoogleBrew`, `newGoogleBrew`, and `readFileMetadata` functons.